.env.master

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Values below are intended to override the same values defined in .env.stable, and are used to do
-# integration testing of in-development components.
+# integration testing of master components.
```

Comment:
It looks like you replaced many instances of level / in-development with master. What was the rationale? Also, it would be good to try to use as much inclusive naming as we can. I know that in many projects we still use `master` for the main branch -- we should fix that like we did in fabric-tna. In the mean time, we can avoid unless we need to explicitly refer to the branch name (like in a git command)

Reply:
The reason is that with this change, we are basically allowing to use of in-development versions in the stable environment. So I think it doesn't make any sense to use devel for this environment anymore. I would use something different to let people know that this env file uses the latest "committed" versions. I agree with the more inclusive terminology, I can use `latest` instead of `master`.

Reply:
+1 for `latest`